ci_test/common_python/tools.py

```diff
@@ -1022,4 +1022,5 @@ def gpus_per_node(lbann):
 def get_distconv_environment():
     # TODO: Use the default halo exchange and shuffle method. See https://github.com/LLNL/lbann/issues/1659
     return {"LBANN_DISTCONV_HALO_EXCHANGE": "AL",
-            "LBANN_DISTCONV_TENSOR_SHUFFLER": "AL"}
+            "LBANN_DISTCONV_TENSOR_SHUFFLER": "AL",
+            "LBANN_KEEP_ERROR_SIGNALS" : "1"}
```

Comment:
Why is this necessary?

Reply:
@tbennun Because most of the DistConv layers will fail when those signals are not kept.  I believe that this is an issue with respect to how data is moved between the DistConv and normal tensors. @benson31 Can you comment more.

Reply:
The short version is that distconv adapters don't maintain various layer invariants (yeah, it's a problem) and this is a workaround to make sure necessary data is kept around.

Reply:
Thanks